### PR TITLE
Rey

### PR DESCRIPTION
**Pull request by Roxxistic**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,22 +15,21 @@ There are two board games that can be played:
...
-
-```bash
-- - - - -
-X X X X X
-- - - - -
-- - - - -
-O O O O O
-- - - - -
-```
+  ```md
```
Comment (Tartori) [inline README.md]: not md syntax

File: README.md [hunk 2]
```diff
@@ -40,33 +39,66 @@ O O O O O
...
-```
-
-```bash
-  X X X X X X
-  X X X X X X
-  X X - - O O
-  O O O O O O
-  O O O O O O
-```
+  ```md
```
Comment (Tartori) [inline README.md]: not md either

File: README.md [hunk 3]
```diff
@@ -40,33 +39,66 @@ O O O O O
...
+    X X X X X
+    X X X X X
+    X X - O O
+    O O O O O
+    O O O O O
+    ```
+  ```md
```
Comment (Tartori) [inline README.md]: neither here

File: README.md [hunk 4]
```diff
@@ -40,33 +39,66 @@ O O O O O
   - stepping left or right 1 field, if this field is occupied by the opponent and the occupying pawn has moved once. (en passant) The opponent's pawn is killed.
 - The game is won by the player who manages first to get a pawn to the last row on the opposite of the board.
 
-#### Strategy
+#### Strategies
```
Comment (Tartori) [inline README.md]: move strategy to top level like pawn chess. it is the main thing in the documentation

File: README.md [hunk 5]
```diff
@@ -78,13 +110,46 @@ TBD
...
 
-TBD
+Time did not allow for any better strategies for playing Alquerque than for a machine that chooses its moves randomly (as in Pawn Chess).
+
+## App Architecture
+
+### Game Engines
 
-Für die Bewertung der Horizontknoten kann statt einer heuristischen Bewertungsfunktion auch ein Monte-Carlo-Test zum Einsatz kommen. Siehe folgenden Artikel:
+### GUI & User Interaction
```
Comment (Tartori) [inline README.md:115]: not really needed

File: README.md [hunk 6]
```diff
@@ -78,13 +110,46 @@ TBD
 
 #### Strategy
 
-TBD
+Time did not allow for any better strategies for playing Alquerque than for a machine that chooses its moves randomly (as in Pawn Chess).
+
+## App Architecture
```
Comment (Tartori) [inline README.md:111]: this whole chapter not needed

File: README.md [hunk 7]
```diff
@@ -78,13 +110,46 @@ TBD
...
+
+### User Interaction
+
 - As a user, interact with the app by entering one of the commands shown at each step.
+
+### Gameplay
+
 - Note that on the board, you as a user are placed on the bottom side while the opponent is placed on the top side.
+
+  ```md
```
Comment (Tartori) [inline README.md]: not markdown

File: controllers/machine_simple_heuristic_actor.py [hunk 8]
```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/python3
+from controllers import BaseMachineActor
+from random import random, choice
+from copy import deepcopy
+
+
+class MachineSimpleHeuristicActor(BaseMachineActor):
+    """
+    This controller takes over during a game play when its the turn
+    of a machine player that follows the MachineStrategies.CLEVER.
```
Comment (Tartori) [inline controllers/machine_simple_heuristic_actor.py]: it is not clear when which actor is used

File: models/machine_strategies.py [hunk 9]
```diff
@@ -7,3 +7,4 @@ class MachineStrategies:
     """
     RANDOM = 0
     CLEVER = 1
+    SIMPLE_HEURISTIC = 2
```
Comment (Tartori) [inline models/machine_strategies.py:10]: not consistent, complex_heursitic missing

File: controllers/machine_complex_heuristic_actor.py [hunk 10]
```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/python3
+from controllers import MachineABPruningActor
+
+
+class MachineComplexHeuristicActor(MachineABPruningActor):
+    """
+    This controller takes over during a game play when its the turn
+    of a machine player that follows the MachineStrategies.CLEVER.
```
Comment (Tartori) [inline controllers/machine_complex_heuristic_actor.py]: Not clear which actor is used by strategy clever

**Review by Tartori — CHANGES_REQUESTED** [review]

File: README.md (hunk 6 above)
Comment (Roxxistic) [reply README.md:111]: i think its required

File: README.md [hunk 11]
```diff
@@ -40,33 +39,66 @@ O O O O O
...
-TBD
+The heuristic evaluation function is based on various considerations:
+- Total pawns:
+  - The more pawns one has compared to the number of pawns the opponent has, the better.
+- Position of one pawn relative to the game board:
+  - The further advanced a pawn, the better.
+  - A pawn that has an unblocked path to the finish line
+    - is great.
+    - is less great, if there is an opponent's pawn in a neighbouring line up ahead and thus could kill our pawn.
+- Position of one pawn relative to our and opponent's other pawns:
```
Comment (Tartori) [inline README.md:98]: not sure if this is extremely relevant. This whole part is already covered by optimizing for more pawns than oponent

**Review by Tartori — APPROVED** [review]

**Pull request merged**